market/src/main/java/university/market/member/service/MemberService.java

```diff
@@ -0,0 +1,52 @@
+package university.market.member.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import university.market.member.domain.MemberVO;
+import university.market.member.exception.LoginException;
+import university.market.member.exception.LoginExceptionType;
+import university.market.member.service.dto.request.JoinRequest;
+import university.market.member.mapper.MemberMapper;
+import university.market.member.service.dto.request.LoginRequest;
+import university.market.member.service.dto.response.LoginResponse;
+import university.market.member.utils.JwtTokenProvider;
+
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+
+    private final MemberMapper memberMapper;
+    private final JwtTokenProvider jwtTokenProvider;
+    private final PasswordEncoder passwordEncoder;
+
+    @Transactional
+    public boolean joinMember(JoinRequest joinRequest) {
+        if(memberMapper.findMemberByEmail(joinRequest.email()) != null) {
```

Comment:
```suggestion
        if (memberMapper.findMemberByEmail(joinRequest.email()) != null) {
```
특별히 문제되지 않는 다면 코드에 IDE에서 제공하는 formatter를 적용하는 것도 좋습니다.